ddtrace/span.py

```diff
@@ -332,7 +332,7 @@ def set_tag(self, key, value=None):
         except Exception:
             log.warning("error setting tag %s, ignoring it", key, exc_info=True)
 
-    def _set_str_tag(self, key, value):
+    def set_str_tag(self, key, value):
```

Comment:
```suggestion
    def set_tag_str(self, key, value):
```

maybe is a little better since we're changing it 😄